### PR TITLE
New package: font-vazir-code-1.1.2

### PR DESCRIPTION
**Pull request by make-file**
This font support of persian/farsi glyphes  in monospaced fonts, which is not found in other already added persian/farsi fonts.

### CODE REVIEW

File: srcpkgs/font-vazir-code/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'font-vazir-code'
+pkgname=font-vazir-code
+version=1.1.2
+revision=1
+create_wrksrc=yes
+depends="font-util"
+short_desc="Persian (farsi) Monospaced Font for coding"
```
Comment (ericonr) [inline srcpkgs/font-vazir-code/template:7]: I don't think it needs the "for coding" part.

Comment (make-file) [reply srcpkgs/font-vazir-code/template]: removed.

File: srcpkgs/font-vazir-code/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'font-vazir-code'
+pkgname=font-vazir-code
+version=1.1.2
+revision=1
+create_wrksrc=yes
+depends="font-util"
+short_desc="Persian (farsi) monospaced font"
+maintainer="mkf <makefile@riseup.net>"
+license="custom:Public Domain"
```
Comment (ericonr) [inline srcpkgs/font-vazir-code/template:9]: So, I'm not totally sure this is the license... According to https://github.com/rastikerdar/vazir-code-font/blob/master/LICENSE , I think we should include the bitstream license as well. Btw, you can just use "Public Domain", no need to make it custom.

Comment (make-file) [reply srcpkgs/font-vazir-code/template]: seems no.

**Pull request closed without merging**